### ISSUE TITLE
Floating Point Precision

### ISSUE THREAD

**Issue by flyncx** [AdlerVaz/Calculator#1]
Hey uh, if I were to calculate `0.1+0.2` it will show `0.30000000000000004`. 
I do think that any calculator's behavior needs to be precise and consistent.
Imagine someone do math with this calculator involving decimals and the result are wrong.

Not to diss about You, I just think this could be better!

**Comment by AdlerVaz** [comment]
yeah i have seen that  ... i think i can fix it ... tysm for that
